cmd/soroban-rpc/main.go

```diff
@@ -283,7 +283,7 @@ func main() {
 			serviceConfig.IngestionTimeout = time.Duration(ingestionTimeoutMinutes) * time.Minute
 			serviceConfig.CoreRequestTimeout = time.Duration(coreTimeoutSeconds) * time.Second
 			serviceConfig.MaxHealthyLedgerLatency = time.Duration(maxHealthyLedgerLatencySeconds) * time.Second
-			daemon.Run(serviceConfig, endpoint, adminEndpoint)
+			daemon.MustNew(serviceConfig, endpoint, adminEndpoint).Run()
```

Comment:
I honestly don't feel comfertable using the function output without testing it.
( yes, I'm aware of the content.. )
But I'm ok if we'll tackle this another time.